.github/workflows/test.yaml

```diff
@@ -130,6 +130,7 @@ jobs:
             --env 'RELATED_IMAGE_POSTGRES_13_GIS_3.1=registry.developers.crunchydata.com/crunchydata/crunchy-postgres-gis:ubi8-13.9-3.1-2' \
             --env 'RELATED_IMAGE_POSTGRES_14=registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.6-2' \
             --env 'RELATED_IMAGE_POSTGRES_14_GIS_3.1=registry.developers.crunchydata.com/crunchydata/crunchy-postgres-gis:ubi8-14.6-3.1-2' \
+            --env 'PGO_FEATURE_GATES=TablespaceVolumes=true' \
```

Comment:
:question: Should this be enabled in `deploy-dev`?

Reply:
Hmmm, I guess so because if you run `deploy-dev` for KUTTL testing (as I do), you'd want that on...

Reply:
Added that